recipes/cysgp4/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "cysgp4" %}
+{% set version = "0.3.0" %}
+{% set sha256 = "24bf9cdc2168c7fc6b6dd89a24849c09560b755d7268e97f6988c08aa09ff84e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py27]
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+
+  host:
+    - openmp 4.0.*  # [osx]
+    - python
+    - pip
+    - setuptools
+    - setuptools_scm
+    - cython
+    - numpy
+
+  run:
+    - python
+    - openmp 4.0.*  # [osx]
+    - {{ pin_compatible('numpy') }}
+
+test:
+  requires:
+    - setuptools
+  imports:
+    - cysgp4
+
+  commands:
+     python -c "import cysgp4; cysgp4.test()"
+
+about:
+  home: https://github.com/bwinkel/{{ name }}
+  license: GPL-3.0
+  license_family: GPL
```

Comment:
On PyPI, this package's license is GPL, while in the repo, the license is Apache. Is there are reason for the disparity of the license types?

Reply:
Actually, three licenses apply (see meta.yaml). The package is a wrapper around a C++ library (Apache license), while my wrapper code is GPL-3. The docs are based on the astropy package template (BSD). I'd consider GPL (my contribution) as the main license for the package, which is why I used this for PyPI. (I'm not sure, if one can specify multiple licenses on PyPI - will try to find out). GitHub unfortunately chooses the first license file it can find and I didn't find out how to change that either. In the [user manual](https://bwinkel.github.io/cysgp4/latest/license.html), it is explained and I'll make sure to update the README.rst in the next release.

Reply:
The license should then likely be something like `GPL-3.0 AND BSD-3-Clause` or, if this package also contains code from the wrapped software, `GPL-3.0 AND BSD-3-Clause AND Apache-2.0`.
Also make sure to include all those licenses in the `license_file` field (as list of filenames).

Reply:
Done! The `license_file` already had all three license files referenced. Sorry, for wasting your time. I appreciate your help!

Reply:
We appreciate the contribution! Hence, it's not time wasted ;).